app/controllers/idv/gpo_controller.rb

```diff
@@ -15,7 +15,7 @@ def index
 
       if current_async_state.none?
         analytics.track_event(
-          Analytics::IDV_GPO_ADDRESS_VISITED,
+          'IdV: USPS address visited',
           letter_already_sent: @presenter.letter_already_sent?,
         )
```

Comment:
this should use the new method, not the string, right?
`analytics.idv_gpo_address_visited(letter_already_sent: @presenter.letter_already_sent?)`

Reply:
Ah yes - updated.